packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -1,31 +1,36 @@
 <template>
   <span ref="cardContainer" :class="['card-container', addClass]">
-    <div v-show="localMinimized" class="morph">
+    <span v-show="localMinimized" class="morph">
```

Comment:
@wxwxwxwx9 how about just shifting this and the root into another container? and then changing the root tag for the other one.

Shouldn't need to change the "normal" view's tags as such

Reply:
Thanks for bringing this up! Initially I went with the 'changing "normal" view's tags' approach because I thought that it would be difficult to rewrite the panel transitions if we did this `shifting this and the root into another container? and then changing the root tag for the other one` using `v-if` / `v-else` for the root nodes. 

Using `v-if` / `v-else` essentially it means that panel ref will be destroyed and the event listeners that we attached to the panel will be gone as well, which would affect our current implementation of panel transition. I thought that it would be difficult to rewrite the panel transition code to fit the new changes (of using `v-if` / `v-else`). 

But as you suggested, it turned out to be relatively straightforward. I've tested the panel transition behaviors and I don't see any regressions so far. The only exception is the behavior as shown in the recording below.

https://user-images.githubusercontent.com/37920115/115955613-3321e180-a52a-11eb-9c23-6fac2f7c2647.mov

As you can see, by minimizing the panel, we essentially "lose" the information of which nested panels were expanded. In other words, every nested panels that were expanded are closed upon minimizing the parent panel. 

I don't think there's a way around this particular regression because when we use `v-if` / `v-else` for rendering, it means that panels are destroyed upon minimizing. Thus, we won't be able to retain the information of which nested panels were expanded; retriever, if there is any, has to re-load as well. But I think this is a minor issue and this particular behavior doesn't seem "wrong" as well, so I think it should be fine to keep it as is.

Reply:
I would imagine this is useful if:
- reader minimizes panel to focus on some other area of the page
- reopens panel, and wants to "resume" where they left off

But this is low priority since it only affects minimization and not panel open / close; We can find a way to keep the second part alive after the first render later on.